Options & Credentials decoding (#655)
mapbox/mapbox-directions-swift: CHANGELOG.md

## v2.3.0

* Added `VisualInstruction.Component.ShieldRepresentation` struct for displaying a highway shield. Added `VisualInstruction.Component.ImageRepresentation.shield` property. ([#644](https://github.com/mapbox/mapbox-directions-swift/pull/644), [#647](https://github.com/mapbox/mapbox-directions-swift/pull/647))

> * Added `DirectionsOptions.init?(url:)`, `RouteOptions.restore(from:)` and `DirectionsOptions.restore(from:)` methods to allow deserializing corresponding options object using appropriate request URL.([#655](https://github.com/mapbox/mapbox-directions-swift/pull/655))

Reviewer: I think this is the form that jazzy requires for autolinking:

```suggestion
* Added `DirectionsOptions(url:)`, `RouteOptions.restore(from:)` and `DirectionsOptions.restore(from:)` methods for deserializing corresponding options object using appropriate request URL. ([#655](https://github.com/mapbox/mapbox-directions-swift/pull/655))
```
